src/app/backend/handler/localehandler.go

```diff
@@ -116,6 +108,17 @@ func (handler *LocaleHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 	if acceptLanguage == "" {
 		acceptLanguage = r.Header.Get("Accept-Language")
 	}
+
+	// TODO(floreks): Remove that once new locale codes are supported by the browsers,
+	// or golang.org/x/text/language map properly.
+	// For backward compatibility only.
+	localeMap := strings.NewReplacer(
+		"zh-CN", "zh-Hans",
```

Comment:
Why `zh-sg` and `zh-hk` mappings are not here?

Reply:
I found that Match returns tag as `zh-Hant-HK` for `zh-HK`, and `zh-Hans-SG` for `zh-SG`, so we don't need to replace them. However we need to replace `zh-CN` and `zh-TW`.